Source/Scene/Cesium3DTileset.js

```diff
@@ -0,0 +1,1821 @@
+         * @see Fog#screenSpaceErrorFactor
+         */
+        this.dynamicScreenSpaceErrorFactor = 4.0;
+
+        /**
+         * A ratio of the tileset's height at which the density starts to falloff. If the camera is below this height the
+         * full computed density is applied, otherwise the density falls off. This has the effect of higher density at
+         * street level views.
+         * <p>
+         * Valid values are between 0.0 and 1.0.
+         * </p>
+         *
+         * @type {Number}
+         * @default 0.25
+         */
+        this.dynamicScreenSpaceErrorHeightFalloff = 0.25;
+
+        this._dynamicScreenSpaceErrorComputedDensity = 0.0; // Updated based on the camera position and direction
+
+        /**
+         * Determines whether the tileset casts or receives shadows from each light source.
+         * <p>
+         * Enabling shadows has a performance impact. A tileset that casts shadows must be rendered twice, once from the camera and again from the light's point of view.
+         * </p>
+         * <p>
+         * Shadows are rendered only when {@link Viewer#shadows} is <code>true</code>.
+         * </p>
+         *
+         * @type {ShadowMode}
+         * @default ShadowMode.ENABLED
+         */
+        this.shadows = defaultValue(options.shadows, ShadowMode.ENABLED);
+
+        /**
+         * Determines if the tileset will be shown.
+         *
+         * @type {Boolean}
+         * @default true
+         */
+        this.show = defaultValue(options.show, true);
+
+        /**
+         * Defines how per-feature colors set from the Cesium API or declarative styling blend with the source colors from
+         * the original feature, e.g. glTF material or per-point color in the tile.
+         *
+         * @type {Cesium3DTileColorBlendMode}
+         * @default Cesium3DTileColorBlendMode.HIGHLIGHT
+         */
+        this.colorBlendMode = Cesium3DTileColorBlendMode.HIGHLIGHT;
+
+        /**
+         * Defines the value used to linearly interpolate between the source color and feature color when the {@link Cesium3DTileset#colorBlendMode} is <code>MIX</code>.
+         * A value of 0.0 results in the source color while a value of 1.0 results in the feature color, with any value in-between
+         * resulting in a mix of the source color and feature color.
+         *
+         * @type {Number}
+         * @default 0.5
+         */
+        this.colorBlendAmount = 0.5;
+
+        /**
+         * This property is for debugging only; it is not optimized for production use.
+         * <p>
+         * Determines if only the tiles from last frame should be used for rendering.  This
+         * effectively "freezes" the tileset to the previous frame so it is possible to zoom
+         * out and see what was rendered.
+         * </p>
+         *
+         * @type {Boolean}
+         * @default false
+         */
+        this.debugFreezeFrame = defaultValue(options.debugFreezeFrame, false);
```

Comment:
Perhaps for both the `options` object and in this class, put everything inside of a `debug` property rather than prefixing everything?